Utils/Functions/System/virtual desktop/script.js

```diff
@@ -0,0 +1,19 @@
+const screenManager = Bean("screenManager");
+
+/**
+ * Obtains the width of user's screen
+ * @returns {number} width of user's screen in pixels
+ */
+// eslint-disable-next-line no-unused-vars
+function getScreenWidth() {
```

Comment:
You need to export this method via `module.getScreenWidth = getScreenWidth;`

Reply:
Please take a look at your changes regarding this comment and the next one again.
Your code now contains only the assignments:

```javascript
module.getScreenWidth = getScreenWidth;
module.getScreenHeight = getScreenHeight;
```

This alone is basically equal to:

```javascript
module.getScreenWidth = undefined;
module.getScreenHeight = undefined;
```

because you have removed the definitions for `getScreenWidth` and `getScreenHeight`.

I recommend you to look at similar scripts to what you want to implement (in this case the [files script](https://github.com/PhoenicisOrg/scripts/blob/master/Utils/Functions/Filesystem/Files/script.js)) and write your code analogous. This helps preventing a lot of questions :)